README.md

```diff
@@ -963,7 +955,7 @@ Keeps value within the range [Min, X, Max] by sort based on integer value (param
 
 Arguments: strings, arrays, or numerics.
 
-Note: From Puppet 6.0.0 this can be done using only built-in functions like this:
+Since Puppet 6.0.0, you can use built-in functions to get the same results:
```

Comment:
`results` is in plural, there is only one result in the example, and there is only one example - I understand the plural refers to the `functions` in plural and that there is actually technically actually are several intermediate results - but I find it confusing as it is really just the single end result that is of interest.